components/FreeRTOS-Libraries-Integration-Tests/config/qualification_wrapper_config.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * ESP32-C3 Featured FreeRTOS IoT Integration V202204.00
+ * Copyright (C) 2022 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * https://www.FreeRTOS.org
+ * https://github.com/FreeRTOS
+ *
+ */
+
+#ifndef QUALIFICATION_WRAPPER_CONFIG_H
+#define QUALIFICATION_WRAPPER_CONFIG_H
+
+/* ESP-IDF sdkconfig include. */
+#include <sdkconfig.h>
```

Comment:
As I alluded to in the GettingStartedGuide.md review - could we not instead just include the sdkconfig into test_execution_config.h and test_param_config.h and set the test configs using the menuconfig configs?

Reply:
IDT modifies test_execution_config.h and test_param_config.h during testing. So we need to adjust the value from these two configuration files to integration test. I also use CONFIG_GRI_RUN_QUALIFICATION_TEST to separate qualication tests & normal demo.